Microsoft.Azure.CosmosRepository/src/Extensions/ParameterRebinder.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
```

Comment:
I like that this was separated into its own file, kudos.

Reply:
👍